modules/develop/pages/produce-data/configure-producers.adoc

```diff
@@ -1,69 +1,95 @@
-is acknowledged by the broker, and is useful when you are working with
-Redpanda clusters.
+communicate their status while transferring data. The `acks` (acknowledgments) property controls the behavior of the producer when a message
+is acknowledged by the broker. A message is considered acknowledged when
+replication of the message to the majority of other brokers responsible for the
+partition in the cluster is complete. Redpanda guarantees data safety by flushing every message to disk before acknowledgement back to clients. 
 
-While it may seem logical that the producer should wait for the broker to
-acknowledge the message in every circumstance, there are cases when you can
-gain more by letting the messages fly to the broker, unconfirmed.
+== Producer optimization strategies
 
-Sometimes trading speed for durability is the best strategy, but sometimes you
-may want it the other way around. A message is considered "acknowledged" when
-replication of the message to the majority of other brokers responsible for the
-partition in the cluster is complete.
+You can optimize for speed (throughput and latency) or safety (durability and
+availability) by adjusting properties. Finding the optimal configuration depends
+on your use case and requires some trial and error. 
 
-== Producer configuration options
+There are many configuration options within Redpanda. The
+configuration options mentioned here work best when combined with other
+broker and consumer configuration options. For details, see
+xref:deploy:deployment-option/self-hosted/manual/node-property-configuration.adoc[Configure Broker Properties]
+and xref:develop:consume-data/consumer-offsets.adoc[Consumer Offsets].
 
-There are a myriad of client configuration options for all components of a
-cluster. Finding the optimal combination of these components depends largely on
-your specific use case. The following sections describe producer configuration
-options, and how to combine these options with other component configurations
-for optimal results.
+=== Optimize for speed
 
-=== Producer acknowledgement settings
+To get data into
+Redpanda as quickly as possible, you can maximize latency and throughput in a variety of ways: 
 
-==== `acks=0`
+* Experiment with <<Producer acknowledgment settings, acks>> settings. The quicker a producer receives a reply from the broker that the
+message has been committed, the sooner it can send the next message, which
+generally results in higher throughput. Hence, if you set `acks=1`, then the
+leader broker does not need to wait for replication to occur, and it can reply
+as soon as it finishes committing the message. This
+can result in less durability overall.
+* Enable <<Write caching, write caching>>, which caches batches of writes in memory until the segment appender chunk is full, instead of flushing for every `acks=all` write. Write caching provides lower latency and higher write throughput at the expense of lower durability guarantees.
```

Comment:
> which caches batches of writes in memory until the segment appender chunk is full, instead of flushing for every `acks=all` write. Write caching provides lower latency and higher write throughput at the expense of lower durability guarantees.

I'd rather rephrase it from the perspective of when it is acked (like in the acks=1) than what it does (how it batches is an implementation detail that doesn't seem to be of interest in this section). For example (please rephrase as you see fit)

> write caching  acknowledges a user request as soon as it is buffered on a majority of nodes and does not wait for it to flush to disk. This gives lower latencies while still ensuring majority of nodes acknowledge the write.

Reply:
Thanks @bharathv! Please confirm if this feature summary is correct, since we'll use it in multiple placses: 

Write caching acknowledges a message as soon as it is buffered on a majority of brokers, without waiting for it to fsync to disk. This provides lower latency while still ensuring that a majority of brokers acknowledge the write.

(I'll edit this page to make it more consistent with use of fsync over flush, message over request, and broker over node.)